test-requirements.txt

```diff
@@ -21,5 +21,6 @@ smmap2
 tornado
 urllib3
 vulners
+ntplib
 pytest-pythonpath
-pycrypto
+pycrypto
```

Comment:
This is odd. I can't detect any change but it still shows a diff on this line. 🤔